module/x/gravity/keeper/genesis.go

```diff
@@ -38,16 +39,20 @@ func InitGenesis(ctx sdk.Context, k Keeper, data types.GenesisState) {
 
 	// reset batch confirmations in state
 	for _, conf := range data.LogicCallConfirms {
+		conf := conf
 		k.SetLogicCallConfirm(ctx, &conf)
 	}
 
 	// reset pool transactions in state
 	for _, tx := range data.UnbatchedTransfers {
-		k.setPoolEntry(ctx, tx)
+		if err := k.setPoolEntry(ctx, tx); err != nil {
+			panic(err)
+		}
 	}
 
 	// reset attestations in state
 	for _, att := range data.Attestations {
+		att := att
```

Comment:
What lint does this re-declaration come from? I'm curious about the motivation. Is it to unalias a pointer?

Reply:
it comes from here https://github.com/kyoh86/scopelint. It checks for unpinned variables, its more of a nit as i haven't seen it cause issues.